docs/explanations/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: page
+title: Explanations
+has_children: true
+---
+
+Here are _understanding-oriented_ explanations.
```

Comment:
Bonus points for linking to the diataxis documentation to describe what each of these sections are for.

Reply:
I can do it for 3, but not for Reference since it's autogenerated.